src/Syncromatics.Clients.WaySine/Models/Sign.cs

```diff
@@ -5,10 +5,19 @@ namespace Syncromatics.Clients.WaySine.Models
 {
     public class Sign
     {
+        [JsonProperty("UtcTimeStamp")]
+        private DateTimeOffset _utcTimeStamp;
+
         [JsonProperty("SignID")]
         public int SignId { get; set; }
-        public DateTimeOffset UtcTimeStamp { get; set; }
-        public DateTime ClientTimeStamp { get; set; }
+
+        // The API gives us the UTC DateTime without any timezone information,
+        // so the deserializer assumes it is in system local time.  This
+        // gets rid of the added timezone.
+        [JsonIgnore]
+        public DateTimeOffset UtcTimeStamp =>
+            _utcTimeStamp.Add(_utcTimeStamp.Offset);
```

Comment:
This will incorrectly change the date represented. The time represented should be the date/time represented in the string, but understood as UTC.

Reply:
I think this actually produces the desired result:
```suggestion
_utcTimeStamp.ToOffset(TimeSpan.Zero).Add(_utcTimeStamp.Offset)
```

Reply:
It is actually the correct moment in time, but yours is still more correct because if we are calling it "UtcTimeStamp" we probably shouldn't be sending anything with an offset.  I'll make the update.